src/items/tools/hammer.js

```diff
@@ -0,0 +1,27 @@
+import Tool from "./tool.js";
+
+export default class Hammer extends Tool {
```

Comment:
When you use a hammer on a bar it should tell you that you need to go to an anvil to do this. Right now it just says the default "this is an invalid crafting combination"

Reply:
Has been implemented with recent commits.